eng/testing/xunit/xunit.console.targets

```diff
@@ -87,11 +87,9 @@
     </ItemGroup>
 
     <!-- Workaround for https://github.com/xunit/xunit/issues/1651 -->
-    <ItemGroup Condition="'$(ArchiveTests)' != 'true'">
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\net452\xunit.runner.utility.net452.dll" />
```

Comment:
This was not a legit workaround -- the binclash here was a symptom of referencing two different runners.  The VS runner was already conditioned out when ArchiveTests is set, so that condition wasn't really doing anything.  
https://github.com/dotnet/runtime/blob/31fa0794d63a41a1b5db50bc4a5e3eadb6a786eb/eng/testing/xunit/xunit.props#L24-L29
Additionally these paths were stale after recent updates changed the folders.

The workaround is still needed - for xunit.abstractions.dll, as mentioned in the linked issue.